HBFA/UefiHostFuzzTestCasePkg/UefiHostFuzzTestCasePkg.dsc

```diff
@@ -219,7 +219,7 @@
    PlatformInitLib|OvmfPkg/Library/PlatformInitLib/PlatformInitLib.inf
    PeilessStartupLib|OvmfPkg/Library/PeilessStartupLib/PeilessStartupLib.inf
    RegisterFilterLib|MdePkg/Library/RegisterFilterLibNull/RegisterFilterLibNull.inf
-   UefiCpuLib|UefiCpuPkg/Library/BaseUefiCpuLib/BaseUefiCpuLib.inf
+   #UefiCpuLib|UefiCpuPkg/Library/BaseUefiCpuLib/BaseUefiCpuLib.inf

```

Comment:
Please remove it

Reply:
Hi @liuw1, this library has been removed from edk2, do we need to keep this line uncommented?